### PR TITLE
Added troubleshooting information to GitHub Apps docs

### PR DESCRIPTION
**Pull request by jrtaylorJH**
## Hey, I just made a Pull Request!

---
'@backstage/docs': patch
---

Added some simple documentation regarding troubleshooting a particularly obtuse error regarding GitHub Apps integrations. Related to multiple open issues:

https://github.com/backstage/backstage/issues/13282
https://github.com/backstage/backstage/issues/19562

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by jrtaylorJH** [comment]
Unclear why docs quality is failing. Line length maybe?

File: docs/integrations/github/github-apps.md
Comment (awanlin) [inline docs/integrations/github/github-apps.md:148]:
```suggestion
is the result of not installing the app in your organization. Even if created via the `backstage-cli`
```

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for this @jrtaylorJH, just a typo to fix. Also, is there any other GitHub documentation we could point to for further details on this?

**Comment by awanlin** [comment]
Just a note for future contributions: `changesets` are created using a command line tool which will create a file you would commit with your PR, but in the case of changes that are only documentation you don't need one. More information on `changesets` here: https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets

**Comment by jrtaylorJH** [comment]
> Just a note for future contributions: `changesets` are created using a command line tool which will create a file you would commit with your PR, but in the case of changes that are only documentation you don't need one. More information on `changesets` here: https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets

Thanks for that info! I'll update my other PR

https://github.com/backstage/backstage/pull/22109

If you wouldn't mind taking a look as well :)

**Comment by jrtaylorJH** [comment]
> Thanks for this @jrtaylorJH, just a typo to fix. Also, is there any other GitHub documentation we could point to for further details on this?

Unfortunately this is one of those that is *incredibly* obtuse. Backstage actually generates the `NotFoundError` by making inferences on the attributes of the app itself, namely "is len of installations list greater than 0", then in turn wraps that error in this "HttpError", so at the end of the day none of the errors actually come from GitHub.

There *is* a guide for installing a GitHub app seen here: https://docs.github.com/en/apps/using-github-apps/installing-your-own-github-app

However, this makes no mention of the "Install" field not appearing due to the lack of "Owner" permissions, which in my opinion is much of the issue.

**Review by awanlin — APPROVED** [review]
Looks great, thanks again!

Just need a 2nd review from one of the Maintainers!

File: docs/integrations/github/github-apps.md
Comment (Zaperex) [inline docs/integrations/github/github-apps.md:150]:
```suggestion
must possess the `Owner` role in the organization to see the `Install` menu under your
```

**Comment by jrtaylorJH** [comment]
@awanlin Just a ping if you're in the business of reviewing PRs today :)

**Comment by awanlin** [comment]
> @awanlin Just a ping if you're in the business of reviewing PRs today :)

Still looks good, I can't merge without another approval from the maintainers so just waiting on that 👍

**Review by Rugvip — APPROVED** [review]
Good bit, thank you! 👍 🎉

**Pull request merged**
